### DISCUSSION TITLE
Cant find (media) in archive

### DISCUSSION THREAD

**Discussion by Urbanscap** [Pustur/whatsapp-chat-parser-website#40] [Q&A]
Every time I upload sample chat zip files, I notice the chat parser is unable to display any media (images, videos, etc).

Instead, it will give a message such as the following: Error: Can't find "00000035-PHOTO-2021-04-08-09-38-45.jpg" in archive

The actual file is included in the zip file and I'm not sure what could be causing the issue. Has anybody else encountered the same problem?

**Comment by Pustur** [comment]
Hi @Urbanscap, please provide the following info to help me debug the issue:

- Are you using your own zip file or the sample provided in the page?
   - If you are using your own, is it coming from whatsapp directly? Or did you modify something yourself? Is it an iPhone or Android export?
- What browser (and version) are you using?
- What OS are you using?

**Reply by Urbanscap** [reply]
Thanks for the response.

1. The file was exported directly from WhatsApp on an iPhone.
2. Firefox ver. 130.0
3. macOS Monterey ver 12.3.1

**Reply by Pustur** [reply]
@Urbanscap This is a recent example of a message with attachment that works for me:

`‎[17.08.2024, 20:14:14] Redacted person: ‎<attached: 00000016-PHOTO-2024-08-17-20-14-14.jpg>`

Could you share a message with an attacment from your .txt file? Maybe there are other formats that are not currently being handled.

**Reply by Urbanscap** [reply]
This is the message format that was exported from whatsapp:

‎[9/4/24, 10:12:46 AM] Redacted: ‎<attached: 00000114-PHOTO-2021-04-09-10-12-46.jpg>

**Reply by Pustur** [reply]
Hi @Urbanscap,

Unfortunately I can't reproduce the issue you're having.
I tried to make a new `_chat.txt` file with the line you sent me, then I created a simple jpg file with the name `00000114-PHOTO-2021-04-09-10-12-46.jpg` and it worked fine.

As last resort, you could email me the zip file that's causing errors, but I know this could contain private information so instead you could create a new chat just for testing and send me that one? (the whole zip file I mean)

PS: I'll leave the test zip file I prepared here in case you want to try that one
[Archive.zip](https://github.com/user-attachments/files/16922280/Archive.zip)
